source/ranges.tex

```diff
@@ -6308,10 +6316,13 @@
       { get<N>(t) } -> convertible_to<const tuple_element_t<N, T>&>;
     };
 
+  template<class T, size_t N>
+  concept @\defexposconcept{returnable-element}@ = is_reference_v<T> || move_constructible<tuple_element_t<N, T>>;
```

Comment:
Oops, I forgot to add an `// exposition only` for this.

Can we add it editorially, seeing as the _`italicized-kebab`_ name clearly signifies that it is supposed to be exposition-only?

Reply:
it's clear that it was intended, thanks!   fixed